webapp/components/Paginate/Paginate.vue

```diff
@@ -1,26 +0,0 @@
-<template>
-  <ds-flex direction="row-reverse">
-    <ds-flex-item width="50px">
-      <ds-button @click="next" :disabled="!hasNext" icon="arrow-right" primary />
-    </ds-flex-item>
-    <ds-flex-item width="50px">
-      <ds-button @click="back" :disabled="!hasPrevious" icon="arrow-left" primary />
-    </ds-flex-item>
-  </ds-flex>
-</template>
-<script>
-export default {
-  props: {
-    hasNext: { type: Boolean, default: false },
-    hasPrevious: { type: Boolean, default: false },
-  },
-  methods: {
-    back() {
-      this.$emit('back')
-    },
-    next() {
-      this.$emit('next')
-    },
-  },
-}
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-15T16:06:35Z" title="Wednesday, January 15th 2020, 5:06:35 pm +01:00">Jan 15, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,26 +1,42 @@
 <template>
-  <ds-flex direction="row-reverse">
-    <ds-flex-item width="50px">
-      <ds-button @click="next" :disabled="!hasNext" icon="arrow-right" primary />
-    </ds-flex-item>
-    <ds-flex-item width="50px">
-      <ds-button @click="back" :disabled="!hasPrevious" icon="arrow-left" primary />
-    </ds-flex-item>
-  </ds-flex>
+  <div class="paginate">
+    <base-button
+      @click="$emit('back')"
+      :disabled="!hasPrevious"
+      icon="arrow-left"
+      circle
+      data-test="previous-button"
+    />
+    <base-button
+      @click="$emit('next')"
+      :disabled="!hasNext"
+      icon="arrow-right"
+      circle
+      data-test="next-button"
+    />
+  </div>
```


I like the new design of the buttons in the `Paginate` component...
I guess you considered renaming it to multi-word following Vue guidelines, but decided not to in this `PR`??
Also, I think the way it is now, it wasn't clear to me that the buttons were disabled until I looked at the design in `admin/users` to see what they look like when there is a `hasNext` or `hasPrevious`... maybe we could just not show them if both those conditions are false? I would be happy to get other opinions/take care of this in a separate `PR`

![Screenshot from 2020-01-15 17-01-59](https://user-images.githubusercontent.com/26943915/72449690-60c19c00-37b9-11ea-8e2f-b7077906e223.png)

Reply:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-01-22T07:41:09Z" title="Wednesday, January 22nd 2020, 8:41:09 am +01:00">Jan 22, 2020</time>_

---


I like the idea of hiding the buttons when there is neither `next` nor `previous`! And yes, I didn't want to take this PR too far. But I'm glad you decided to change the name in your commit. :wink: